Multisig/UI/Transactions/TransactionViewModel/TransactionViewModel.swift

```diff
@@ -41,7 +44,10 @@ class TransactionViewModel: Identifiable {
 
     init(_ tx: Transaction, _ safe: SafeStatusRequest.Response) {
         id = UUID()
+        hash = tx.transactionHash?.description
         date = tx.executionDate ?? tx.submissionDate ?? tx.modified
+        createdDate = tx.submissionDate.map { Self.dateFormatter.string(from: $0) } ?? ""
```

Comment:
we need to clarify this, because the creation date is not returned in the transaction body

Reply:
what do you mean? we have submission date in transaction body

Reply:
I mean that creation date is not the same as submission date

Reply:
clarified, indeed it is the creation date! ✅